### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Mre55**
### In this PR:

-  Initialise React app.
- Add React Redux
- Structure the application files using a "feature folder" approach and use the ducks pattern for your Redux files.
- Add React Router and set two <Route>s and <Link>s for the app's navigation.
- Set Books to the default view.

### CODE REVIEW

File: src/components/Books/Books.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <br />
+        <button type="button">Remove</button>
+      </li>
+      <li className={styles.book_section}>
+        Sample Book Three
+        <br />
+        <br />
+        <button type="button">Remove</button>
+      </li>
+    </ul>
```
Comment (Mihndim2020) [inline src/components/Books/Books.js]: - [ ] As it is a requirement for this project, please kindly add a `BookList` component that;
 - Renders a list of books using the `Book component`,
 - Renders a form by using the `CreateNewBook` component.

File: src/components/Books/Books.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <label htmlFor="book-dropdown" className="grid-item">
+        <select id="book-dropdown">
+          <option value="" disabled selected>Category</option>
+          <option value="action">Action</option>
+          <option value="science-fiction">Science Fiction</option>
+          <option value="economy">Economy</option>
+        </select>
+      </label>
+      <button type="button" className="grid-item">ADD BOOK</button>
+    </form>
```
Comment (Mihndim2020) [inline src/components/Books/Books.js]: - [ ] As it is a requirement for this project, please kindly add a `CreateNewBook` component, that;
- Renders a form with inputs for the title and the author of the book,
- Has a button to submit(the button wouldn't add the book *yet*, this is just a placeholder for now).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Mre55, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: No `Linter` errors.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by D4nielJ — APPROVED** [review]
## Hello @Mre55, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**